src/view_text.cpp

```diff
@@ -212,9 +220,18 @@ void View::DrawLyricString(DeviceContext *dc, std::wstring str, int staffSize)
         dc->SetFont(&vrvTxt);
         std::wstring str;
         str.push_back(VRV_TEXT_E551);
-        dc->DrawText(UTF16to8(str), str);
+        if (params) {
+            dc->DrawText(UTF16to8(str), str, params->m_x, params->m_y, params->m_width, params->m_height);
+        }
+        else {
+            dc->DrawText(UTF16to8(str), str);
+        }
         dc->ResetFont();
     }
+
+    if (!wroteText && params) {
+        dc->DrawText("", L"", params->m_x, params->m_y, params->m_width, params->m_height);
+    }
```

Comment:
Could you comment on this?

Reply:
If there's no text but a zone is specified (providing the height and width) then draw a blank space.

Reply:
OK. This comment in the code would be useful - it has to be clear that this is only with facs/zone

Reply:
Got it!